data.json

```diff
@@ -21,7 +21,8 @@
     {"name": "Harbin", "address": "4801 Saint-Laurent"},
     {"name": "Boucherie Lawrence's", "address": "5237 Saint-Laurent"},
     {"name": "B&M", "address": "120 Saint-Viateur O"},
-    {"name": "MAÏS", "address": "5439 Saint-Laurent"}
+    {"name": "MAÏS", "address": "5439 Saint-Laurent"},
+    {"name": "Euro Déli Batory", "address": "115 Saint-Viateur O"}
```

Comment:
Ummm everywhere on their website is Euro-Deli Batory... maybe the vitrine stuff is only for the OQLF... I'm confused too now, do as you want 🙈